mache/cime_machine_config/config_machines.xml

```diff
@@ -1433,7 +1443,7 @@
         <command name="reset"></command>
         <command name="load">PrgEnv-cray</command>
         <command name="load">craype-accel-amd-gfx90a</command>
-        <command name="load">rocm/5.1.0</command>
+        <command name="load">rocm/5.4.0</command>
```

Comment:
I notice we have `frontier_crayclanggpu_mpich.yaml`, is that file referring to the `frontier-scream-gpu` machine? However, when I check `frontier_crayclanggpu_mpich.yaml`, I see that `rocm/5.1.0` has already changed to `rocm/5.4.0`, so we don't need to change anything here. Cue confusion!